apps/nextjs-website/src/components/Feature.tsx

```diff
@@ -13,7 +14,7 @@ const makeFeatureProps = ({
   items: items.map((item) => ({
     stackIcon: {
       ...(isValidMuiIcon(item.icon) && {
-        startIcon: <Icon>{formatValidMuiIcon(item.icon)}</Icon>,
+        icon: item.icon as keyof typeof MuiIcons,
```

Comment:
You don't need the cast if `isValidMuiIcon` returns a [type predicate](https://www.typescriptlang.org/docs/handbook/2/narrowing.html#using-type-predicates). You can just change [isValidMuiIcon](https://github.com/pagopa/b2b-portals/blob/93e0be0/apps/nextjs-website/src/components/Icons/index.ts#L4-L5) as follows:

```typescript
export const isValidMuiIcon = (iconName?: string | null): iconName is (keyof typeof MuiIcons) =>
  !!iconName && iconName in MuiIcons;
```